test.js

```diff
@@ -0,0 +1,252 @@
+import test from 'node:test'
```

Comment:
Nice! Let's see whether `node:test` is ready for using in other of our repositories too 👍🏻